resources/public/api/conf/2.0/application.yaml

```diff
@@ -0,0 +1,10973 @@
+                <p class="code--block">
+                  lisaManagerReferenceNumber: 123456<br>
+                  accountId: 1234567890
+                </p>
+              </td>
+              <td>
+                <p>HTTP status: <code class="code--slim">400 (Bad Request)</code></p>
+        <pre class="code--block">
+        {
+          "code": "BAD_REQUEST",
+          "message": "Enter lisaManagerReferenceNumber in the correct format, like Z1234"
+        }
+        </pre>
+              </td>
+            </tr>
+            <tr>
+              <td>
+                <p>Request with a valid LISA Manager reference number, but an invalid account ID</p>
+                <p class="code--block">
+                  lisaManagerReferenceNumber: <a href="#section/Testing">Use your test user profile</a><br>
+                  accountId: 1234%3D5678
+                </p>
+              </td>
+              <td>
+                <p>HTTP status: <code class="code--slim">400 (Bad Request)</code></p>
+        <pre class="code--block">
+        {
+          "code": "BAD_REQUEST",
+          "message": "Enter accountId in the correct format, like ABC12345"
+        }
+        </pre>
+              </td>
+            </tr>
+            <tr>
+                <td>
+                    <p>Request containing an account ID that does not exist</p>
+                    <p class="code--block">
+                        lisaManagerReferenceNumber: <a href="#section/Testing">Use your test user profile</a><br>
+                        accountId: 0000000404
+                    </p>
+                </td>
+                <td>
+                  <p>HTTP status: <code class="code--slim">404 (Not found)</code></p>
+        <pre class="code--block">
+        {
+          "code": "INVESTOR_ACCOUNTID_NOT_FOUND",
+          "message": "Enter a real accountId"
+        }
+        </pre>
+                </td>
+            </tr>
+            <tr>
+              <td>
+                <p>Request with an invalid 'Accept' header</p>
+                <p class="code--block">
+                  lisaManagerReferenceNumber: <a href="#section/Testing">Use your test user profile</a>
+                  <br>accountId: 1234567890<br>
+                  <br>
+                  Accept: application/vnd.hmrc.1.0
+                </p>
+              </td>
+              <td>
+                <p>HTTP status: <code class="code--slim">404 (Not Found)</code></p>
+        <pre class="code--block">
+        {
+          "code": "MATCHING_RESOURCE_NOT_FOUND",
+          "message": "A resource with the name in the request can not be found in the API"
+        }
+        </pre>
+              </td>
+            </tr>
+          </tbody>
+        </table>
+      operationId: getAccountDetails
+      parameters:
+      - $ref: '#/components/parameters/acceptHeader'
+      - $ref: '#/components/parameters/contentTypeHeader'
```

Comment:
This does not require a Content-Type header (the RAML is incorrect) as this is a retrieval endpoint.

<img width="659" alt="Screenshot 2023-02-10 at 09 04 41" src="https://user-images.githubusercontent.com/63714619/218049688-945cfa47-fde0-46a9-a894-6ddf23ea9410.png">

Please correct this at least in this PR by removing this reference. Thanks.

<img width="1127" alt="Screenshot 2023-02-10 at 08 53 51" src="https://user-images.githubusercontent.com/63714619/218049998-53eef8cc-13d1-4dde-9ff6-f74f82b1c514.png">